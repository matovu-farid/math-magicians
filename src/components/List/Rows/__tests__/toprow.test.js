import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import TopSymbolRow from '../toprow';

it.skip('should have 3 buttons', () => {
  render(<TopSymbolRow />);
  const listItems = screen.queryAllByRole('listitem');

  expect(listItems.length).toBe(3);
});
