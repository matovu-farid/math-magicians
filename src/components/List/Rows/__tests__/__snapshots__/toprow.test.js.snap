// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul
  className="flex"
>
  <li>
    <button
      className="tile"
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
  </li>
  <li>
    <button
      className="tile"
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
  </li>
  <li>
    <button
      className="tile"
      onClick={[Function]}
      type="button"
    >
      %
    </button>
  </li>
</ul>
`;
